frontend/src/components/filters/FiltersFrequent.js

```diff
@@ -57,9 +50,7 @@ class FiltersFrequent extends PureComponent {
    * @summary ToDo: Describe the method
    * @todo Write the documentation
    */
-  handleClickOutside = () => {
-    this.outsideClick();
-  };
+  handleClickOutside = () => this.outsideClick();
```

Comment:
Why not move logic from `outsideClick` here and remove it completely ?

Reply:
Good point. I'll consider it. Thanks!